rocAL_pybind/amd/rocal/fn.py

```diff
@@ -1136,3 +1136,34 @@ def tensor_mul_scalar_float(*inputs, scalar=1.0, rocal_tensor_output_type=types.
     kwargs_pybind = {"input_image0": inputs[0], "is_output": False, "rocal_tensor_output_type": rocal_tensor_output_type, "scalar": scalar}
     tensor_mul_scalar_float = b.tensorMulScalar(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
     return tensor_mul_scalar_float
+
+# //TODO: To modify the comments wrt earlier augmentations
+def nonsilent_region(*inputs, rocal_tensor_output_type = types.FLOAT, bytes_per_sample_hint = [0], cutoff_db = -60, reference_power = 0.0, reset_interval = 8192, seed = -1, window_length = 2048):
```

Comment:
is this code correct? NSR outputs are of type int in latest changes
```
rocal_tensor_output_type = types.FLOAT
```

Reply:
Remove that variable, it is not used